test/test_ops.py

```diff
@@ -1535,6 +1535,7 @@ class TestRefsOpsInfo(TestCase):
         '_refs.broadcast_tensors',
         '_refs.nn.functional.tanhshrink',
         '_refs.swap_axes',
+        '_refs.unflatten',
```

Comment:
Why is this necessary? Now `unflatten` is an ATen op, right?

Reply:
Correct, removed.